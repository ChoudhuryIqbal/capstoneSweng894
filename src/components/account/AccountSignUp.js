import React from 'react'
import { View, Text, Platform } from 'react-native'
import { InputCustom, InputNumberCustom } from '../common'
import CustomInputStyles from '../styles/CustomInputStyles'
import { CustomPicker } from 'react-native-custom-picker'

/**
 * Account Log-In/Sign-Up user & password fields 
 * 
 * @param {*} props 
 */
const  AccountSignUp = (props) => {
    return(
        <View>
            <View style={{flex: 1, flexDirection: 'row'}}>
                <InputCustom
                    placeholder="i.e. William"
                    label="First Name: "     
                    value={props.firstNameValue}
                    onChangeText={props.firstNameChge}
                    error={props.errorFirstName}
                    inputStyle = {Platform.Os === 'ios' ? CustomInputStyles.inputStyleLeft : CustomInputStyles.inputStyleAndroid}
                    containerStyle = {CustomInputStyles.containerStyleLeft}      
                    labelStyle = {CustomInputStyles.labelStyle}
                    errorStyle = {Platform.Os === 'ios' ? CustomInputStyles.error : CustomInputStyles.errorAndroid} 
                />
                 <InputCustom
                    placeholder="i.e. Jackson"
                    label={"Last Name: "}
                    value={props.lastNameValue}
                    onChangeText={props.lastNameChge}
                    error={props.errorLastName}
                    inputStyle = {Platform.Os === 'ios' ? CustomInputStyles.inputStyleOsRight : CustomInputStyles.inputStyleAndroid}
                    containerStyle = {CustomInputStyles.containerStyleRight}      
                    labelStyle = {CustomInputStyles.labelStyleRight}
                    errorStyle = {Platform.Os === 'ios' ? CustomInputStyles.error : CustomInputStyles.errorAndroid} 
                />
            </View>
            <View style={{flex: 1, flexDirection: 'row'}}>
                <InputCustom
                    placeholder="i.e. 1987-01-01"
                    label={"Date of Birth: "}
                    value={props.dobValue}
                    onChangeText={props.dobChge}
                    error={props.errordob}
                    inputStyle = {Platform.Os === 'ios' ? CustomInputStyles.inputStyleOsRight : CustomInputStyles.inputStyleAndroid}
                    containerStyle = {CustomInputStyles.containerStyleRight}      
                    labelStyle = {CustomInputStyles.labelStyleRight}
                    errorStyle = {Platform.Os === 'ios' ? CustomInputStyles.error : CustomInputStyles.errorAndroid} 
               />
            </View>
            <View style={{flex: 1, flexDirection: 'row'}}>
                <InputCustom
                    placeholder="i.e. e@dummy.com"
                    label={"Email: "}
                    value={props.emailValue}
                    onChangeText={props.emailOnChge}
                    error={props.errorEmail}   
                    inputStyle = {Platform.Os === 'ios' ? CustomInputStyles.inputStyleLeft : CustomInputStyles.inputStyleAndroid}
                    containerStyle = {CustomInputStyles.containerStyleLeft}       
                    labelStyle = {CustomInputStyles.labelStyle}
                    errorStyle = {Platform.Os === 'ios' ? CustomInputStyles.error : CustomInputStyles.errorAndroid} 
                />
                 <InputCustom
                    secureTextEntry
                    placeholder="Password"
                    label={"Password:"}
                    value={props.passwordValue}
                    onChangeText={props.passwordOnChge}
                    error={props.errorPassword} 
                    inputStyle = {Platform.Os === 'ios' ? CustomInputStyles.inputStyleOsRight : CustomInputStyles.inputStyleAndroid}
                    containerStyle = {CustomInputStyles.containerStyleRight}      
                    labelStyle = {CustomInputStyles.labelStyleRight}
                    errorStyle = {Platform.Os === 'ios' ? CustomInputStyles.error : CustomInputStyles.errorAndroid} 
                />
            </View>
            <View style={{flex: 1, flexDirection: 'row'}}>
                <InputCustom
                    placeholder="i.e. 123-123-1234"
                    label="Telephone: " 
                    value={props.telephoneValue}
                    onChangeText={props.telephoneOnChge}
                    error={props.errorTelephone}        
                    inputStyle = {Platform.Os === 'ios' ? CustomInputStyles.inputStyleLeft : CustomInputStyles.inputStyleAndroid}
                    containerStyle = {CustomInputStyles.containerStyleLeft}       
                    labelStyle = {CustomInputStyles.labelStyle}
                    errorStyle = {Platform.Os === 'ios' ? CustomInputStyles.error : CustomInputStyles.errorAndroid} 
                />
                <View style={CustomInputStyles.containerStyleRight}>
                    <Text>{"Gender: "}</Text>
                    <CustomPicker
                        placeholder={"i.e. Male"}
                        options={props.genderItem}
                        onValueChange={props.genderOnChge}
                        defaultValue={"i.e. Male"}
                        value={props.genderValue}
                    />
                    <Text style={Platform.Os === 'ios' ? CustomInputStyles.error : CustomInputStyles.errorAndroid}>{props.errorGender}</Text>
                </View>
            </View>
            
            <View style={{flex: 1, flexDirection: 'row'}}>
                <InputCustom
                    placeholder="i.e. 100 North Street"
                    label={"Address: "}
                    value={props.streetValue}
                    onChangeText={props.streetOnChge}
                    error={props.errorStreet}
                    inputStyle = {Platform.Os === 'ios' ? CustomInputStyles.inputStyleLeft : CustomInputStyles.inputStyleAndroid}
                    containerStyle = {CustomInputStyles.containerStyleLeft}      
                    labelStyle = {CustomInputStyles.labelStyle}
                    errorStyle = {Platform.Os === 'ios' ? CustomInputStyles.error : CustomInputStyles.errorAndroid} 

                />
                 <InputCustom
                    placeholder="i.e. Philadelphia"
                    label={"City:"}
                    value={props.cityValue}
                    onChangeText={props.cityOnChge}
                    error={props.errorCity}
                    inputStyle = {Platform.Os === 'ios' ? CustomInputStyles.inputStyleLeft : CustomInputStyles.inputStyleAndroid}
                    containerStyle = {CustomInputStyles.containerStyleRight}      
                    labelStyle = {CustomInputStyles.labelStyleRight}
                    errorStyle = {Platform.Os === 'ios' ? CustomInputStyles.error : CustomInputStyles.errorAndroid} 

                />
            </View>
            <View style={{flex: 1, flexDirection: 'row'}}>
                <InputCustom
                    placeholder="i.e. PA"
                    label={"State: "}
                    value={props.stateValue}
                    onChangeText={props.stateOnChge}
                    error={props.errorState}
                    inputStyle = {Platform.Os === 'ios' ? CustomInputStyles.inputStyleOsRight : CustomInputStyles.inputStyleAndroid}
                    containerStyle = {CustomInputStyles.containerStyleLeft}      
                    labelStyle = {CustomInputStyles.labelStyle}
                    errorStyle = {Platform.Os === 'ios' ? CustomInputStyles.error : CustomInputStyles.errorAndroid} 

                />
                <InputNumberCustom
                    placeholder="i.e. 19475"
                    label="Zipcode: "
                    value={props.zip}
                    onChangeText={props.onZipChge}
                    error={props.errorZip}
                    inputStyle = {Platform.Os === 'ios' ? CustomInputStyles.inputStyleOsRight : CustomInputStyles.inputStyleAndroid}
                    containerStyle = {CustomInputStyles.containerStyleLeft}      
                    labelStyle = {CustomInputStyles.labelStyle}
                    errorStyle = {Platform.Os === 'ios' ? CustomInputStyles.error : CustomInputStyles.errorAndroid} 
                />
            </View>
        </View>
    )
}

export default AccountSignUp