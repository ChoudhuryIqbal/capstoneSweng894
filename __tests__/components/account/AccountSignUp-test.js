import React from 'react'
import AccountSignUp from '../../../src/components/account/AccountSignUp'
import {create} from 'react-test-renderer';

describe('Account Sign Up correctly renders', () => {
    test('Matches the snapshot', () => {
        const accountSignUp = create(
            <AccountSignUp
                firstNameValue={'Test'}
                firstNameChge={() => {}}
                errorFirstName={'Test'}
                lastNameValue={'Test'}
                lastNameChge={() => {}}
                errorLastName={'Test'}
                emailValue={'Test'}
                emailOnChge={() => {}}
                errorEmail={'Test'}
                passwordValue={'Test'}
                passwordOnChge={() => {}}
                errorPassword={'Test'}
                telephoneValue={'Test'}
                telephoneOnChge={() => {}}
                errorTelephone={'Test'} 
                genderOnChge={() => {}}
                genderItem={[]}
                genderValue={'Test'}
                errorGender={'Test'}
                dobValue={'Test'}
                dobChge={() => {}}
                errordob={'Test'}
                streetValue={'Test'}
                streetOnChge={() => {}}
                errorStreet={'Test'}
                cityValue={'Test'}
                cityOnChge={() => {}}
                errorCity={'Test'}
                stateValue={'Test'}
                stateOnChge={() => {}}
                errorState={'Test'}
                zip={'Test'}
                onZipChge={() => {}}
                errorZip={'Test'} 
            /> 
        )
        expect(accountSignUp.toJSON()).toMatchSnapshot();
    })
})