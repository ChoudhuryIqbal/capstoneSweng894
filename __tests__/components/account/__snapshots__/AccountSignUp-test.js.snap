// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Account Sign Up correctly renders Matches the snapshot 1`] = `
<View>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "row",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "flex-start",
          "backgroundColor": "#ffffff",
          "display": "flex",
          "flex": 1,
          "flexBasis": 90,
          "flexDirection": "column",
          "justifyContent": "center",
          "margin": 5,
        }
      }
    >
      <Text
        style={
          Object {
            "flex": 1,
            "fontSize": 16,
            "paddingBottom": 10,
          }
        }
      >
        First Name: 
      </Text>
      <TextInput
        allowFontScaling={true}
        autoCorrect={false}
        onChangeText={[Function]}
        placeholder="i.e. William"
        rejectResponderTermination={true}
        style={
          Object {
            "backgroundColor": "#ffffff",
            "color": "#000",
            "elevation": 1,
            "flex": 2,
            "fontSize": 18,
            "lineHeight": 3,
            "paddingBottom": 20,
            "paddingLeft": 5,
            "paddingRight": 5,
            "shadowOffset": Object {
              "height": 10,
              "width": 5,
            },
            "shadowOpacity": 1,
            "shadowRadius": 5,
          }
        }
        underlineColorAndroid="transparent"
        underlineColorAnroid="transparent"
        value="Test"
      />
      <Text
        style={
          Object {
            "bottom": 0,
            "color": "red",
            "elevation": 1,
            "fontSize": 10,
            "justifyContent": "center",
            "marginTop": 10,
            "position": "absolute",
            "textAlign": "center",
          }
        }
      >
        Test
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "flex-start",
          "backgroundColor": "#ffffff",
          "display": "flex",
          "flex": 1,
          "flexBasis": 90,
          "flexDirection": "column",
          "justifyContent": "center",
          "margin": 5,
        }
      }
    >
      <Text
        style={
          Object {
            "flex": 1,
            "fontSize": 16,
            "paddingBottom": 10,
          }
        }
      >
        Last Name: 
      </Text>
      <TextInput
        allowFontScaling={true}
        autoCorrect={false}
        onChangeText={[Function]}
        placeholder="i.e. Jackson"
        rejectResponderTermination={true}
        style={
          Object {
            "backgroundColor": "#ffffff",
            "color": "#000",
            "elevation": 1,
            "flex": 2,
            "fontSize": 18,
            "lineHeight": 3,
            "paddingBottom": 20,
            "paddingLeft": 5,
            "paddingRight": 5,
            "shadowOffset": Object {
              "height": 10,
              "width": 5,
            },
            "shadowOpacity": 1,
            "shadowRadius": 5,
          }
        }
        underlineColorAndroid="transparent"
        underlineColorAnroid="transparent"
        value="Test"
      />
      <Text
        style={
          Object {
            "bottom": 0,
            "color": "red",
            "elevation": 1,
            "fontSize": 10,
            "justifyContent": "center",
            "marginTop": 10,
            "position": "absolute",
            "textAlign": "center",
          }
        }
      >
        Test
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "row",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "flex-start",
          "backgroundColor": "#ffffff",
          "display": "flex",
          "flex": 1,
          "flexBasis": 90,
          "flexDirection": "column",
          "justifyContent": "center",
          "margin": 5,
        }
      }
    >
      <Text
        style={
          Object {
            "flex": 1,
            "fontSize": 16,
            "paddingBottom": 10,
          }
        }
      >
        Date of Birth: 
      </Text>
      <TextInput
        allowFontScaling={true}
        autoCorrect={false}
        onChangeText={[Function]}
        placeholder="i.e. 1987-01-01"
        rejectResponderTermination={true}
        style={
          Object {
            "backgroundColor": "#ffffff",
            "color": "#000",
            "elevation": 1,
            "flex": 2,
            "fontSize": 18,
            "lineHeight": 3,
            "paddingBottom": 20,
            "paddingLeft": 5,
            "paddingRight": 5,
            "shadowOffset": Object {
              "height": 10,
              "width": 5,
            },
            "shadowOpacity": 1,
            "shadowRadius": 5,
          }
        }
        underlineColorAndroid="transparent"
        underlineColorAnroid="transparent"
        value="Test"
      />
      <Text
        style={
          Object {
            "bottom": 0,
            "color": "red",
            "elevation": 1,
            "fontSize": 10,
            "justifyContent": "center",
            "marginTop": 10,
            "position": "absolute",
            "textAlign": "center",
          }
        }
      >
        Test
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "row",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "flex-start",
          "backgroundColor": "#ffffff",
          "display": "flex",
          "flex": 1,
          "flexBasis": 90,
          "flexDirection": "column",
          "justifyContent": "center",
          "margin": 5,
        }
      }
    >
      <Text
        style={
          Object {
            "flex": 1,
            "fontSize": 16,
            "paddingBottom": 10,
          }
        }
      >
        Email: 
      </Text>
      <TextInput
        allowFontScaling={true}
        autoCorrect={false}
        onChangeText={[Function]}
        placeholder="i.e. e@dummy.com"
        rejectResponderTermination={true}
        style={
          Object {
            "backgroundColor": "#ffffff",
            "color": "#000",
            "elevation": 1,
            "flex": 2,
            "fontSize": 18,
            "lineHeight": 3,
            "paddingBottom": 20,
            "paddingLeft": 5,
            "paddingRight": 5,
            "shadowOffset": Object {
              "height": 10,
              "width": 5,
            },
            "shadowOpacity": 1,
            "shadowRadius": 5,
          }
        }
        underlineColorAndroid="transparent"
        underlineColorAnroid="transparent"
        value="Test"
      />
      <Text
        style={
          Object {
            "bottom": 0,
            "color": "red",
            "elevation": 1,
            "fontSize": 10,
            "justifyContent": "center",
            "marginTop": 10,
            "position": "absolute",
            "textAlign": "center",
          }
        }
      >
        Test
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "flex-start",
          "backgroundColor": "#ffffff",
          "display": "flex",
          "flex": 1,
          "flexBasis": 90,
          "flexDirection": "column",
          "justifyContent": "center",
          "margin": 5,
        }
      }
    >
      <Text
        style={
          Object {
            "flex": 1,
            "fontSize": 16,
            "paddingBottom": 10,
          }
        }
      >
        Password:
      </Text>
      <TextInput
        allowFontScaling={true}
        autoCorrect={false}
        onChangeText={[Function]}
        placeholder="Password"
        rejectResponderTermination={true}
        secureTextEntry={true}
        style={
          Object {
            "backgroundColor": "#ffffff",
            "color": "#000",
            "elevation": 1,
            "flex": 2,
            "fontSize": 18,
            "lineHeight": 3,
            "paddingBottom": 20,
            "paddingLeft": 5,
            "paddingRight": 5,
            "shadowOffset": Object {
              "height": 10,
              "width": 5,
            },
            "shadowOpacity": 1,
            "shadowRadius": 5,
          }
        }
        underlineColorAndroid="transparent"
        underlineColorAnroid="transparent"
        value="Test"
      />
      <Text
        style={
          Object {
            "bottom": 0,
            "color": "red",
            "elevation": 1,
            "fontSize": 10,
            "justifyContent": "center",
            "marginTop": 10,
            "position": "absolute",
            "textAlign": "center",
          }
        }
      >
        Test
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "row",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "flex-start",
          "backgroundColor": "#ffffff",
          "display": "flex",
          "flex": 1,
          "flexBasis": 90,
          "flexDirection": "column",
          "justifyContent": "center",
          "margin": 5,
        }
      }
    >
      <Text
        style={
          Object {
            "flex": 1,
            "fontSize": 16,
            "paddingBottom": 10,
          }
        }
      >
        Telephone: 
      </Text>
      <TextInput
        allowFontScaling={true}
        autoCorrect={false}
        onChangeText={[Function]}
        placeholder="i.e. 123-123-1234"
        rejectResponderTermination={true}
        style={
          Object {
            "backgroundColor": "#ffffff",
            "color": "#000",
            "elevation": 1,
            "flex": 2,
            "fontSize": 18,
            "lineHeight": 3,
            "paddingBottom": 20,
            "paddingLeft": 5,
            "paddingRight": 5,
            "shadowOffset": Object {
              "height": 10,
              "width": 5,
            },
            "shadowOpacity": 1,
            "shadowRadius": 5,
          }
        }
        underlineColorAndroid="transparent"
        underlineColorAnroid="transparent"
        value="Test"
      />
      <Text
        style={
          Object {
            "bottom": 0,
            "color": "red",
            "elevation": 1,
            "fontSize": 10,
            "justifyContent": "center",
            "marginTop": 10,
            "position": "absolute",
            "textAlign": "center",
          }
        }
      >
        Test
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "flex-start",
          "backgroundColor": "#ffffff",
          "display": "flex",
          "flex": 1,
          "flexBasis": 90,
          "flexDirection": "column",
          "justifyContent": "center",
          "margin": 5,
        }
      }
    >
      <Text>
        Gender: 
      </Text>
      <View>
        <View
          accessible={true}
          focusable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
            }
          }
        >
          <View>
            <View>
              <View
                style={
                  Object {
                    "alignItems": "flex-start",
                    "backgroundColor": "#ffffff",
                    "display": "flex",
                    "flex": 1,
                    "flexBasis": 90,
                    "flexDirection": "column",
                    "justifyContent": "center",
                    "margin": 5,
                  }
                }
              >
                <View
                  style={
                    Object {
                      "backgroundColor": "#ffffff",
                      "color": "#000",
                      "elevation": 1,
                      "flex": 2,
                      "fontSize": 18,
                      "lineHeight": 3,
                      "paddingBottom": 20,
                      "paddingLeft": 5,
                      "paddingRight": 5,
                      "shadowOffset": Object {
                        "height": 10,
                        "width": 5,
                      },
                      "shadowOpacity": 1,
                      "shadowRadius": 5,
                    }
                  }
                >
                  <Text>
                    Test
                  </Text>
                  <Text />
                </View>
              </View>
            </View>
          </View>
        </View>
        <Modal
          animationType="none"
          hardwareAccelerated={false}
          onRequestClose={[Function]}
          transparent={true}
          visible={false}
        >
          <View
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Array [
                Object {
                  "backgroundColor": "rgba(0,0,0,0.4)",
                  "flex": 1,
                },
                undefined,
              ]
            }
          >
            <View
              style={
                Array [
                  Object {
                    "flex": 1,
                    "justifyContent": "center",
                    "paddingHorizontal": 20,
                  },
                ]
              }
            >
              <View
                style={
                  Array [
                    Object {
                      "backgroundColor": "white",
                      "maxHeight": 1274,
                    },
                    undefined,
                  ]
                }
              >
                <RCTScrollView>
                  <View />
                </RCTScrollView>
              </View>
            </View>
          </View>
        </Modal>
      </View>
      <Text
        style={
          Object {
            "bottom": 0,
            "color": "red",
            "elevation": 1,
            "fontSize": 10,
            "justifyContent": "center",
            "marginTop": 10,
            "position": "absolute",
            "textAlign": "center",
          }
        }
      >
        Test
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "row",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "flex-start",
          "backgroundColor": "#ffffff",
          "display": "flex",
          "flex": 1,
          "flexBasis": 90,
          "flexDirection": "column",
          "justifyContent": "center",
          "margin": 5,
        }
      }
    >
      <Text
        style={
          Object {
            "flex": 1,
            "fontSize": 16,
            "paddingBottom": 10,
          }
        }
      >
        Address: 
      </Text>
      <TextInput
        allowFontScaling={true}
        autoCorrect={false}
        onChangeText={[Function]}
        placeholder="i.e. 100 North Street"
        rejectResponderTermination={true}
        style={
          Object {
            "backgroundColor": "#ffffff",
            "color": "#000",
            "elevation": 1,
            "flex": 2,
            "fontSize": 18,
            "lineHeight": 3,
            "paddingBottom": 20,
            "paddingLeft": 5,
            "paddingRight": 5,
            "shadowOffset": Object {
              "height": 10,
              "width": 5,
            },
            "shadowOpacity": 1,
            "shadowRadius": 5,
          }
        }
        underlineColorAndroid="transparent"
        underlineColorAnroid="transparent"
        value="Test"
      />
      <Text
        style={
          Object {
            "bottom": 0,
            "color": "red",
            "elevation": 1,
            "fontSize": 10,
            "justifyContent": "center",
            "marginTop": 10,
            "position": "absolute",
            "textAlign": "center",
          }
        }
      >
        Test
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "flex-start",
          "backgroundColor": "#ffffff",
          "display": "flex",
          "flex": 1,
          "flexBasis": 90,
          "flexDirection": "column",
          "justifyContent": "center",
          "margin": 5,
        }
      }
    >
      <Text
        style={
          Object {
            "flex": 1,
            "fontSize": 16,
            "paddingBottom": 10,
          }
        }
      >
        City:
      </Text>
      <TextInput
        allowFontScaling={true}
        autoCorrect={false}
        onChangeText={[Function]}
        placeholder="i.e. Philadelphia"
        rejectResponderTermination={true}
        style={
          Object {
            "backgroundColor": "#ffffff",
            "color": "#000",
            "elevation": 1,
            "flex": 2,
            "fontSize": 18,
            "lineHeight": 3,
            "paddingBottom": 20,
            "paddingLeft": 5,
            "paddingRight": 5,
            "shadowOffset": Object {
              "height": 10,
              "width": 5,
            },
            "shadowOpacity": 1,
            "shadowRadius": 5,
          }
        }
        underlineColorAndroid="transparent"
        underlineColorAnroid="transparent"
        value="Test"
      />
      <Text
        style={
          Object {
            "bottom": 0,
            "color": "red",
            "elevation": 1,
            "fontSize": 10,
            "justifyContent": "center",
            "marginTop": 10,
            "position": "absolute",
            "textAlign": "center",
          }
        }
      >
        Test
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "row",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "flex-start",
          "backgroundColor": "#ffffff",
          "display": "flex",
          "flex": 1,
          "flexBasis": 90,
          "flexDirection": "column",
          "justifyContent": "center",
          "margin": 5,
        }
      }
    >
      <Text
        style={
          Object {
            "flex": 1,
            "fontSize": 16,
            "paddingBottom": 10,
          }
        }
      >
        State: 
      </Text>
      <TextInput
        allowFontScaling={true}
        autoCorrect={false}
        onChangeText={[Function]}
        placeholder="i.e. PA"
        rejectResponderTermination={true}
        style={
          Object {
            "backgroundColor": "#ffffff",
            "color": "#000",
            "elevation": 1,
            "flex": 2,
            "fontSize": 18,
            "lineHeight": 3,
            "paddingBottom": 20,
            "paddingLeft": 5,
            "paddingRight": 5,
            "shadowOffset": Object {
              "height": 10,
              "width": 5,
            },
            "shadowOpacity": 1,
            "shadowRadius": 5,
          }
        }
        underlineColorAndroid="transparent"
        underlineColorAnroid="transparent"
        value="Test"
      />
      <Text
        style={
          Object {
            "bottom": 0,
            "color": "red",
            "elevation": 1,
            "fontSize": 10,
            "justifyContent": "center",
            "marginTop": 10,
            "position": "absolute",
            "textAlign": "center",
          }
        }
      >
        Test
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "flex-start",
          "backgroundColor": "#ffffff",
          "display": "flex",
          "flex": 1,
          "flexBasis": 90,
          "flexDirection": "column",
          "justifyContent": "center",
          "margin": 5,
        }
      }
    >
      <Text
        style={
          Object {
            "flex": 1,
            "fontSize": 16,
            "paddingBottom": 10,
          }
        }
      >
        Zipcode: 
      </Text>
      <TextInput
        allowFontScaling={true}
        autoCorrect={false}
        keyboardType="numbers-and-punctuation"
        onChangeText={[Function]}
        placeholder="i.e. 19475"
        rejectResponderTermination={true}
        style={
          Object {
            "backgroundColor": "#ffffff",
            "color": "#000",
            "elevation": 1,
            "flex": 2,
            "fontSize": 18,
            "lineHeight": 3,
            "paddingBottom": 20,
            "paddingLeft": 5,
            "paddingRight": 5,
            "shadowOffset": Object {
              "height": 10,
              "width": 5,
            },
            "shadowOpacity": 1,
            "shadowRadius": 5,
          }
        }
        underlineColorAndroid="transparent"
        underlineColorAnroid="transparent"
        value="Test"
      />
      <Text
        style={
          Object {
            "bottom": 0,
            "color": "red",
            "elevation": 1,
            "fontSize": 10,
            "justifyContent": "center",
            "marginTop": 10,
            "position": "absolute",
            "textAlign": "center",
          }
        }
      >
        Test
      </Text>
    </View>
  </View>
</View>
`;
